### PR TITLE
Review

### PR DESCRIPTION
**Pull request by dzgierski19**

### CODE REVIEW

File: isValueEven.js [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+  return false;
 }
+
+isInputNumber = (input) => {
+  if (typeof input !== "number" || isNaN(input) === true) {
+    throw new Error("You must type number");
+  }
+};
+
+isInputAnInteger = (input) => {
```
Comment (mwakulinski) [inline isValueEven.js]: See why we couldn't check numbers other than integers? We can check if 3.3425 is even or not. (We both know it is not) But we can still do it :)

File: isValueEven.js [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+  if (Number.isInteger(input) === false) {
+    throw new Error("You must not type an integer");
```
Comment (mwakulinski) [inline isValueEven.js]: Btw, this error message is incorrect :) you check for one thing and write another :)

File: isValueEven.js [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+  }
+};
+
+isInputZero = (input) => {
+  if (input === 0) {
+    throw new Error("You must not type 0");
+  }
+};
```
Comment (mwakulinski) [inline isValueEven.js]: We still can check if 0 is even or not :) So this error throw is unnecessary

**Review by mwakulinski — CHANGES_REQUESTED** [review]
Nice try but you are a little bit overeager :)

File: isValueEven.js (hunk 3 above)
Comment (dzgierski19) [reply isValueEven.js]: validation deleted

File: isValueEven.js (hunk 2 above)
Comment (dzgierski19) [reply isValueEven.js]: validation deleted

**Review by mwakulinski — APPROVED** [review]
Nice job 💪

**Pull request merged**